gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -7275,6 +7275,9 @@ static PyObject* GemRB_IsValidStoreItem(PyObject * /*self*/, PyObject* args)
 	}
 
 	StoreActionFlags ret = store->AcceptableItemType( item->ItemType, Flags, type == 0 || type == 2 );
+	if (!actor->GetThiefLevel()) {
+		ret &= ~StoreActionFlags::Steal;
+	}
```

Comment:
Are you sure this is needed at all — why would only thieves be allowed to steal?
If we really need the check, we should check against the pick pocket skill, since bards also get it.

Reply:
you're right about pickpocketing (how do you check that?)

in BG2 at least only pickpockets can shoplift. Haven't checked others yet.

Reply:
`actor->GetSkill(IE_PICKPOCKET)`

Reply:
I've verified this for PST and BG1 as well.